jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/execution/steps/MessageEmitterTest.kt

```diff
@@ -110,6 +110,24 @@ class MessageEmitterTest {
         }
     }
 
+    @Test
+    fun exceptionsWithoutAMessageGivesType() {
```

Comment:
should we also spit out a stack trace with no message? otherwise if we get a bug report it'll probably be hard to track down

Reply:
Good idea